dotcom-rendering/src/components/ImageComponent.tsx

```diff
@@ -319,13 +329,28 @@ export const ImageComponent = ({
 				{!!title && (
 					<ImageTitle title={title} role={role} palette={palette} />
 				)}
+				{parseInt(imageWidth, 10) > 620 &&
```

Comment:
This check is used in three places and it would be beneficial to have a method like:

```ts
const isWideEnoughForLightbox = (width: number) => width > 620;
```